src/be_txn.erl

```diff
@@ -214,4 +238,34 @@ to_json(blockchain_txn_payment_v2, T, _Ledger) ->
       <<"payments">> => [PaymentJson(Payment) || Payment <- blockchain_txn_payment_v2:payments(T)],
       <<"fee" >> => blockchain_txn_payment_v2:fee(T),
       <<"nonce">> => blockchain_txn_payment_v2:nonce(T),
-      <<"signature">> => ?BIN_TO_B64(blockchain_txn_payment_v2:signature(T)) }.
+      <<"signature">> => ?BIN_TO_B64(blockchain_txn_payment_v2:signature(T)) };
+to_json(blockchain_txn_state_channel_open_v1, T, _Ledger) ->
+    #{<<"id">> => ?BIN_TO_B64(blockchain_txn_state_channel_open_v1:id(T)),
+      <<"owner">> => ?BIN_TO_B58(blockchain_txn_state_channel_open_v1:owner(T)),
+      <<"oui">> => blockchain_txn_state_channel_open_v1:oui(T),
+      <<"fee" >> => blockchain_txn_state_channel_open_v1:fee(T),
+      <<"nonce">> => blockchain_txn_state_channel_open_v1:nonce(T),
+      <<"expire_within">> => blockchain_txn_state_channel_open_v1:expire_within(T),
+      <<"signature">> => ?BIN_TO_B64(blockchain_txn_state_channel_open_v1:signature(T)) };
+to_json(blockchain_txn_state_channel_close_v1, T, _Ledger) ->
+
+    SummaryJson = fun(Summary) ->
+                          #{ <<"client_pubkeybin">> => ?BIN_TO_B58(blockchain_state_channel_summary_v1:client_pubkeybin(Summary)),
```

Comment:
Don't call it pubkey_bin.. Nothing else has type info in it like this.. it's the `client`, right? or is there something specific that we need to know here

Reply:
Renamed 8bec535